demisto_sdk/scripts/validate_content_path.py

```diff
@@ -0,0 +1,432 @@
+).difference(
+    (
+        "Packs",
+        "BaseContents",
+        "BaseNodes",
+        "BasePlaybooks",
+        "BaseScripts",
+        "TestScripts",
+        "CommandOrScripts",
+    )
+)
+
+DEPTH_ONE_FOLDERS_ALLOWED_TO_CONTAIN_FILES = frozenset(
+    (
+        PLAYBOOKS_DIR,
+        TEST_PLAYBOOKS_DIR,
+        REPORTS_DIR,
+        DASHBOARDS_DIR,
+        INCIDENT_FIELDS_DIR,
+        INCIDENT_TYPES_DIR,
+        INDICATOR_FIELDS_DIR,
+        INDICATOR_TYPES_DIR,
+        GENERIC_TYPES_DIR,
+        GENERIC_MODULES_DIR,
+        GENERIC_DEFINITIONS_DIR,
+        LAYOUTS_DIR,
+        CLASSIFIERS_DIR,
+        MAPPERS_DIR,
+        CONNECTIONS_DIR,
+        RELEASE_NOTES_DIR,
+        DOC_FILES_DIR,
+        JOBS_DIR,
+        PRE_PROCESS_RULES_DIR,
+        LISTS_DIR,
+        PARSING_RULES_DIR,
+        MODELING_RULES_DIR,
+        CORRELATION_RULES_DIR,
+        XSIAM_DASHBOARDS_DIR,
+        XSIAM_REPORTS_DIR,
+        TRIGGER_DIR,
+        WIDGETS_DIR,
+        WIZARDS_DIR,
+        LAYOUT_RULES_DIR,
+        *TESTS_AND_DOC_DIRECTORIES,
+    )
+)
+
+DIRS_ALLOWING_SPACE_IN_FILENAMES = frozenset(
+    TESTS_AND_DOC_DIRECTORIES + [TEST_PLAYBOOKS_DIR]
+)
+app = typer.Typer()
+
+
+class InvalidPathException(Exception, ABC):
+    message: ClassVar[str]
+
+
+class SpacesInFileNameError(InvalidPathException):
+    message = "file name contains spaces"
+
+
+class InvalidDepthZeroFile(InvalidPathException):
+    message = "The file cannot be saved direclty under the pack folder."
+
+
+class InvalidDepthOneFolder(InvalidPathException):
+    message = "The name of the first level folder under the pack is not allowed."
+
+
+class InvalidDepthOneFile(InvalidPathException):
+    message = "The folder containing this file cannot directly contain files. Add another folder under it."
+
+
+class InvalidLayoutFileName(InvalidPathException):
+    message = "The Layout folder can only contain json files, with names starting with `layout-` or `layoutscontainer-`"
+
+
+class InvalidIntegrationScriptFileName(InvalidPathException):
+    message = "This file's name must start with the name of its parent folder."
+
+
+class InvalidIntegrationScriptFileType(InvalidPathException):
+    message = "This file type is not allowed under this folder."
+
+
+class InvalidIntegrationScriptMarkdownFileName(InvalidPathException):
+    message = (
+        "This file's name must either be (parent folder)_description.md, or README.md"
+    )
+
+
+class InvalidCommandExampleFile(InvalidPathException):
+    message = "This file's name must be command_examples"
+
+
+class ExemptedPath(Exception, ABC):
+    message: ClassVar[str]
+
+
+class PathOutsidePacks(ExemptedPath):
+    message = "Path is not under Packs"
+
+
+class PathIsFolder(ExemptedPath):
+    message = "Path is to a folder, these are not validated."
+
+
+class PathUnderDeprecatedContent(ExemptedPath):
+    message = "Path under DeprecatedContent, these are not validated."
```

Comment:
Same thing here:
Path under DeprecatedContent is not validated.

Reply:
```suggestion
    message = "Paths under DeprecatedContent are not validated."
```